README.md

```diff
@@ -269,6 +269,21 @@ export default class App {
 }
 ```
 
+#### Decorators
+
+The `@provide` decorator lets you inject a Redux instance more succinctly:
+
+```js
+import { provide } from 'redux/react';
+
+@provide(redux)
+export default class App {
+  render() {
+    return <CounterApp />;
```

Comment:
I kept the example brief for focus and there's a very similar example with `<Provider>` right above this one anyway... which also doesn't import CounterApp for that matter.